common/utils.js

```diff
@@ -581,6 +581,54 @@
         }
 
         /**
+         * @param {ERMrest.Tuple[]} tuples - array of tuples
+         * @param {ERMrest.ReferenceColumn[]} columns - array of column names
+         * @return {Object[]} array of row value arrays [{isHTML: boolean, value: v}, ...]
+         */
+        function getRowValuesFromTupleData(tuples, columns) {
```

Comment:
You can change it to "raw" or leave it as it is. Just add a comment that emphasis on the fact that it returns _raw_ values.